app/models/work.rb

```diff
@@ -0,0 +1,32 @@
+class Work < ApplicationRecord
+
+  # Validations
+  validates :title, presence: true
+  validates :category, presence: true
+  validates :creator, presence: true
+  validates :publication_year, presence: true
+  validates :description, presence: true
+
+  # Relationships
+  has_many :votes, dependent: :destroy
+  has_many :users, :through => :votes
+
+  # Methods
+  def self.spotlight
+    return spotlight_media = Work.order("votes_count DESC").first
```

Comment:
It's a little cleaner to use the keyword arguments instead of a string here here:

```suggestion
    return spotlight_media = Work.order(votes_count: :desc).first
```